templates/geonetwork/ogc-api-records/ogc-api-records-deployment.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and .Values.georchestra.webapps.geonetwork.enabled .Values.georchestra.webapps.geonetwork.ogc_api_records.enabled -}}
+{{- if and .Values.georchestra.webapps.geonetwork.enabled $webapp.enabled -}}
```

Comment:
you're using webapp before defining it, here, no ?

Reply:
resolved